Mathlib/Data/Complex/Exponential.lean

```diff
@@ -202,6 +202,7 @@ noncomputable def expMonoidHom : MonoidHom (Multiplicative ℂ) ℂ :=
     map_one' := by simp,
     map_mul' := by simp [exp_add] }
 
+set_option linter.deprecated false in
```

Comment:
Is it possible to rather fix the proof instead?

Reply:
I tried once and failed; I tried again and now it worked: fixed.